services/shhext/chat/db/db.go

```diff
@@ -0,0 +1,244 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	sqlite "github.com/mutecomm/go-sqlcipher" // We require go sqlcipher that overrides default implementation
+	"github.com/status-im/migrate"
+	"github.com/status-im/migrate/database/sqlcipher"
+	"github.com/status-im/migrate/source/go_bindata"
+	"github.com/status-im/status-go/services/shhext/chat/db/migrations"
+)
+
+const exportDB = "SELECT sqlcipher_export('newdb')"
+
+// The default number of kdf iterations in sqlcipher (from version 3.0.0)
+// https://github.com/sqlcipher/sqlcipher/blob/fda4c68bb474da7e955be07a2b807bda1bb19bd2/CHANGELOG.md#300---2013-11-05
+// https://www.zetetic.net/sqlcipher/sqlcipher-api/#kdf_iter
+const defaultKdfIterationsNumber = 64000
+
+// The reduced number of kdf iterations (for performance reasons) which is
+// currently used for derivation of the database key
+// https://github.com/status-im/status-go/pull/1343
+// https://notes.status.im/i8Y_l7ccTiOYq09HVgoFwA
+const KdfIterationsNumber = 3200
```

Comment:
Do we use it anywhere?

Reply:
yes, it is used in this namespace, in one of the migrations and when we open the db